src/SFA.DAS.EmployerFinance.MessageHandlers.UnitTests/EventHandlers/ExpireFundsCommandHandlerTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Moq;
+using NServiceBus;
+using NUnit.Framework;
+using SFA.DAS.EmployerFinance.Data;
+using SFA.DAS.EmployerFinance.MessageHandlers.CommandHandlers;
+using SFA.DAS.EmployerFinance.Messages.Commands;
+using SFA.DAS.EmployerFinance.Models.ExpiredFunds;
+using SFA.DAS.EmployerFinance.Models.Levy;
+using SFA.DAS.EmployerFinance.Models.Payments;
+using SFA.DAS.EmployerFinance.Types.Models;
+using SFA.DAS.Testing;
+
+namespace SFA.DAS.EmployerFinance.MessageHandlers.UnitTests.EventHandlers
+{
+    [TestFixture]
+    public class ExpireAccountFundsCommandHandlerTests : FluentTest<ExpireAccountFundsCommandHandlerTestsFixture>
+    {
+        [Test]
+        public Task Handle_WhenHandlingExpireAccountFundsCommand_ThenShouldGetTheFundsIn()
```

Comment:
The repository call to get the funds in is an implementation detail and doesn't need to be asserted. Similarly with the `Handle_WhenHandlingExpireAccountFundsCommand_ThenShouldGetTheFundsOut` & `Handle_WhenHandlingExpireAccountFundsCommand_ThenShouldGetTheExistingExpiredFunds` tests so these can be dropped.